source/documentation/metrics-install-guide/Upgrading_Metrics_Store.html.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Replacing fluentd with Rsyslog
+---
+
+# Chapter 5: Upgrading Metrics from 4.2
+
+Since version 4.3 Rsyslog is used to gather metrics from oVirt hosts (replacing fluentd).
+
+Rsyslog is used to collect Engine, VDSM logs and the Collectd metrics, parse the data and add additional metadata and send the data to the remote metrics store.
+
+
+
+## pre-req
+
+* Ovirt Metrics 4.3 with working metrics OpenShift deployment.
+* Ovirt host 4.3 which was previously upgraded from 4.2.
+
+
+
+## Procedure
```

Comment:
This paragraph should be replaced by the installation page.
https://ovirt.org/documentation/metrics-install-guide/metrics_store_installation_guide.html